index.js

```diff
@@ -35,7 +15,7 @@ module.exports = function(options){
       opts.data = file.data;
     }
 
-    file.path = handleExtension(file.path, opts);
+    file.path = gutil.replaceExtension(file.path, opts.client ? '.js' : '.html');
```

Comment:
would like all the `gutil` stuff at the top being assigned to variables because it is deprecated and each method will be replaced by a different dep